apps/test-suite/tests/SQLite.ts

```diff
@@ -114,46 +118,66 @@ CREATE TABLE IF NOT EXISTS test (id INTEGER PRIMARY KEY NOT NULL, name VARCHAR(6
       expect(error).toBeNull();
+
+      await db.closeAsync();
     });
   });
 
   describe('File system tests', () => {
     beforeAll(async () => {
-      await FS.deleteAsync(FS.documentDirectory + 'SQLite', { idempotent: true });
-      await FS.makeDirectoryAsync(FS.documentDirectory + 'SQLite', { intermediates: true });
+      if (process.env.EXPO_OS !== 'web') {
+        await FS.deleteAsync(FS.documentDirectory + 'SQLite', { idempotent: true });
+        await FS.makeDirectoryAsync(FS.documentDirectory + 'SQLite', { intermediates: true });
+      }
     });
 
-    it('should work with a downloaded .db file', async () => {
-      const asset = await Asset.fromModule(require('../assets/asset-db.db')).downloadAsync();
-      await FS.copyAsync({
-        from: asset.localUri,
-        to: `${FS.documentDirectory}SQLite/downloaded.db`,
-      });
+    nativeIt(
+      'should work with a downloaded .db file',
+      async () => {
+        const asset = await Asset.fromModule(require('../assets/asset-db.db')).downloadAsync();
+        await FS.copyAsync({
+          from: asset.localUri,
+          to: `${FS.documentDirectory}SQLite/downloaded.db`,
+        });
 
-      const db = await SQLite.openDatabaseAsync('downloaded.db');
-      const results = await db.getAllAsync<UserEntity>('SELECT * FROM users');
-      expect(results.length).toEqual(3);
-      expect(results[0].j).toBeCloseTo(23.4);
-      await db.closeAsync();
-    }, 30000);
+        const db = await SQLite.openDatabaseAsync('downloaded.db');
+        const results = await db.getAllAsync<UserEntity>('SELECT * FROM users');
+        expect(results.length).toEqual(3);
+        expect(results[0].j).toBeCloseTo(23.4);
+        await db.closeAsync();
+      },
+      30000
+    );
 
     it('should create and delete a database in file system', async () => {
-      const db = await SQLite.openDatabaseAsync('test.db');
+      let db = await SQLite.openDatabaseAsync('test.db');
       await db.execAsync(`
 DROP TABLE IF EXISTS users;
 CREATE TABLE IF NOT EXISTS users (user_id INTEGER PRIMARY KEY NOT NULL, name VARCHAR(64), k INT, j REAL);
+INSERT INTO users (name, k, j) VALUES ('Tim Duncan', 1, 23.4);
 `);
+      const results = await db.getAllAsync<UserEntity>('SELECT * FROM users');
+      expect(results.length).toBe(1);
       await db.closeAsync();
 
-      let fileInfo = await FS.getInfoAsync(`${FS.documentDirectory}SQLite/test.db`);
-      expect(fileInfo.exists).toBeTruthy();
+      // Double check whether the data is persisted
+      db = await SQLite.openDatabaseAsync('test.db');
+      expect((await db.getAllAsync<UserEntity>('SELECT * FROM users')).length).toBe(1);
+      await db.closeAsync();
 
       await SQLite.deleteDatabaseAsync('test.db');
-      fileInfo = await FS.getInfoAsync(`${FS.documentDirectory}SQLite/test.db`);
-      expect(fileInfo.exists).toBeFalsy();
+
+      db = await SQLite.openDatabaseAsync('test.db');
```

Comment:
i want to test `SQLite.openDatabaseAsync` and prevent using expo-file-system. this test is refactored as
- open db and create a table and some data
- open db again and check the previous data is persisted
- call `SQLite.openDatabaseAsync`
- open db and check if not data there